saleor/graphql/shop/types.py

```diff
@@ -128,6 +129,10 @@ class Shop(graphene.ObjectType):
                 "A language code to return the translation for."
             ),
         ),
+        in_shipping_zones=graphene.Boolean(
+            description="Only countries that have shipping zones assigned",
```

Comment:
I think we can document all cases as each has a different result, for example: "When `true`, return X; when `false`, return Y; when the argument is not provided (`null`), return Z."